repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -528,3 +528,13 @@ export const addFlatFareProductIfNotPresent = (): void => {
         }
     });
 };
+
+export const checkAndRepickService = (): void => {
+    cy.get('body').then(($body) => {
+        if ($body.text().includes('Tcannot create a return for this service')) {
+            cy.log('There is a problem, repicking service');
```

Comment:
The log message should include what the problem is, so something like "Trying to use a Service that cannot have a return product created for, trying another service"